imaids/models.py

```diff
@@ -1675,7 +1675,7 @@ def __init__(
                 gap, period_length)
 
         block_len = (
-            period_length/2 - pole_length - longitudinal_distance)
+            period_length/2 - pole_length - 2*longitudinal_distance)
```

Comment:
This issue was previously corrected for the Cassette class but not here, indeed. I added a commit to make this correction on the other hybrid model as well.